rosdep/python.yaml

```diff
@@ -3694,6 +3694,13 @@ python-pymavlink:
 python-pymodbus:
   debian: [python-pymodbus]
   ubuntu: [python-pymodbus]
+python-pymodbus-pip:
```

Comment:
do you want to add the python3 key?

In that case, please use `python3-pymodbus-pip`

Reply:
Would it be alright to **also** add the python 3 key? In order to allow both older ROS distributions to use this too.

Reply:
> Would it be alright to also add the python 3 key? In order to allow both older ROS distributions to use this too.

yes!